charts/ziti-controller/templates/configmap.yaml

```diff
@@ -23,7 +23,7 @@ data:
 
   zitiLogin: |-
     #!/usr/bin/env bash
-    ziti edge login "${ZITI_MGMT_API}" --yes \
```

Comment:
stop accepting unverifiable mgmt api server certs because it must be verifiable by --ca to avoid sending the admin password to a malicious intermediary